xdsl/interactive/app.py

```diff
@@ -314,8 +341,10 @@ def update_current_module(self) -> None:
             )
             pipeline.apply(ctx, module)
             self.current_module = module
+            self.update_number_of_input_ops(input_text)
```

Comment:
it's a shame to reparse and compute everything when we already have the input module a few lines above, could you please change the code so that it doesn't redo this computation?